helm/grafana/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+description: Grafana instance for kube-prometheus
+engine: gotpl
+maintainers:
+- email: weiwei.inf@gmail.com
```

Comment:
Does this mean that you can dedicate some time to maintain this chart? That would be great of course, we'll take any help we can. I just want to make sure that is actually your intention.